blockchain/stake/staketx.go

```diff
@@ -757,8 +757,7 @@ func CheckSSGen(tx *wire.MsgTx) error {
 	// Ensure that the first output is an OP_RETURN push.
 	zeroethOutputVersion := tx.TxOut[0].Version
 	zeroethOutputScript := tx.TxOut[0].PkScript
-	if txscript.GetScriptClass(zeroethOutputVersion, zeroethOutputScript) !=
-		txscript.NullDataTy {
+	if !standalone.IsNullDataScript(zeroethOutputVersion, zeroethOutputScript) {
```

Comment:
This is not used anywhere else outside of the `stake` package and it is consensus.  I suspect it should be an unexported function here in the `stake` package, but I could be convinced otherwise with a good reason.

Reply:
Yeah it shouldn't be exported, `txscript` already has an identical exported call.

Reply:
Actually it's going to have to be exported still because `staketx_test.go` uses a different package for tests.